assets/sass/vendor/_mdc-dialog.scss

```
@@ -27,17 +74,74 @@
 		padding: 0;
 	}
 
+	.mdc-dialog__dependencies {
+		margin: 1em 0;
+		@extend %label-small;
+
+		strong {
+			font-weight: $fw-medium;
+		}
+	}
+
 	.mdc-dialog__provides {
 		overflow: initial;
-		padding: 20px 16px;
+		padding: 10px 0;
+
+		& + .mdc-dialog__dependencies {
+			margin: 0.5em 0 0;
+		}
 	}
 
-	.mdc-dialog__dependencies {
-		margin: 1em 0;
+	.mdc-list {
+
+		&:first-child:last-child {
+			padding: 0;
+		}
+
+		&.mdc-list--underlined {
+
+			.mdc-list-item {
+				@extend %label-small;
+				height: unset;
+				min-height: 32px;
+				padding-left: 10px;
+
+				&::before {
+					background-color: $c-neutral-n-900;
+					border-radius: 50%;
+					content: " ";
+					height: 4px;
+					left: 0;
+					position: absolute;
+					top: 50%;
+					transform: translateY(-50%);
+					width: 4px;
+				}
+			}
+		}
 	}
 
 	.mdc-dialog__actions {
-		justify-content: space-between;
-		margin: 1em 0;
+		column-gap: 10px;
+		margin: 2em 0 0;
+		min-height: 40px;
+		padding: 0;
+	}
+
+	/** Different Size Custom Style **/
```

Extremely nitpicky, but while we're at it, let's use a single line comment.

```suggestion
	// Size variants
```